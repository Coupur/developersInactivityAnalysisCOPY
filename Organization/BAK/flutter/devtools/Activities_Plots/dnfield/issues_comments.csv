id,date,creator_login
481911351,2019-04-10 23:38:54,dnfield
482171880,2019-04-11 15:51:13,dnfield
482294326,2019-04-11 20:14:37,dnfield
