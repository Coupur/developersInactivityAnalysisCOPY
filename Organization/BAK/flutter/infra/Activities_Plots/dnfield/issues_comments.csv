id,date,creator_login
503777651,2019-06-19 23:02:20,dnfield
504489572,2019-06-21 16:34:39,dnfield
492809374,2019-05-15 20:22:40,dnfield
482728957,2019-04-12 21:28:31,dnfield
479088673,2019-04-02 16:45:51,dnfield
475408625,2019-03-21 21:25:21,dnfield
475453243,2019-03-22 00:35:22,dnfield
464269048,2019-02-16 01:23:48,dnfield
