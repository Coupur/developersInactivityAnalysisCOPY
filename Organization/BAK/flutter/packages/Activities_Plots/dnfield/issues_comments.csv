id,date,creator_login
489238323,2019-05-03 21:00:19,dnfield
488176473,2019-05-01 02:15:54,dnfield
488177968,2019-05-01 02:27:44,dnfield
456680030,2019-01-23 05:55:50,dnfield
431165998,2018-10-18 21:14:52,dnfield
431190297,2018-10-18 22:52:27,dnfield
431209707,2018-10-19 00:49:12,dnfield
431909053,2018-10-22 17:36:02,dnfield
436521475,2018-11-07 06:46:00,dnfield
453664253,2019-01-11 21:35:14,dnfield
456695971,2019-01-23 07:17:05,dnfield
457355340,2019-01-24 21:02:58,dnfield
