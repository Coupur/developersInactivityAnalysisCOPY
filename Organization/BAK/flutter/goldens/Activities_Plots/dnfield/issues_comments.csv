id,date,creator_login
486423652,2019-04-24 20:59:14,dnfield
481774335,2019-04-10 16:57:26,dnfield
452857227,2019-01-09 21:12:53,dnfield
452877249,2019-01-09 21:37:08,dnfield
452883378,2019-01-09 21:55:55,dnfield
452883584,2019-01-09 21:56:30,dnfield
452883798,2019-01-09 21:57:11,dnfield
